### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by slhuilli1** [zealdocs/zeal#1506] [General]
Hi,

I suggest you (because I use it very frequently) to add a new doc : the usagge of UTF-8 icons with html entities, html numeric code (&...;) and the image

Thanks

**Comment by SovaPtica** [comment]
Hello, please add the C++ oneAPI documentation.

**Comment by trollixx** [comment]
Docsets are not part of this projects. All docsets currently available inside Zeal are generously provided by [Dash](https://kapeli.com/dash).

Many docsets can be found on the Internet, for example:

- ~[Feeds](http://zealusercontributions.now.sh/) for Dash user contributed docsets.~ Zeal shows these now.
- [Third Party Resources](https://github.com/zealdocs/zeal/wiki/Third-Party-Resources#docsets) wiki page.

You can also generate your own docsets following [this guide](https://kapeli.com/docsets).
